### PR TITLE
Added async scaling routes

### PR DESCRIPTION
**Pull request by ewoutp**
See subject

### CODE REVIEW

**Comment by ewoutp** [comment]
@zyndiecate RFR

File: scale_down.go [hunk 1]
```diff
@@ -35,3 +35,18 @@ func (c *Client) ScaleDownComponent(app, component string, count int) error {
 
 	return nil
 }
+
+func (c *Client) ScaleDownComponentAsync(app, component string, count int) (*Task, error) {
```
Comment (zyndiecate) [inline scale_down.go]: `app` -> `service`

**Comment by zyndiecate** [comment]
LGTM

File: scale_down.go (hunk 1 above)
Comment (ewoutp) [inline scale_down.go]: Done

**Pull request merged**